domains/arifintajul4.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "arifintajul4",
+    "email": "arifintajul4@gmail.com"
+  },
+  "record": {
+    "RECORDTYPE": "tajul.my.id"
```

Comment:
RECORDTYPE should be an actual record type and not a placeholder.

Please refer to the docs: https://docs.is-a.dev/